packages/editor-ui/src/App.vue

```diff
@@ -193,7 +195,7 @@ export default mixins(newVersions, userHelpers).extend({
 		this.loading = false;
 
 		this.trackPage();
-		this.$externalHooks().run('app.mount');
+		void this.externalHooks.run('app.mount');
```

Comment:
to suppress missing `await` warning, I just saw the same in a lot of places and followed the pattern

Reply:
we recently enabled stricter linting rules on promises and async functions. if a function is async, it should be awaited, and if we don't want/can't await, then there should be a `void` to avoid having a unhandled-rejection errors accidentally breaking unrelated code.
I'll send a PR to make `externalHooks.run` fire-and-forget, ans we can then remove all these `void`s.

Reply:
I see. Thanks for the explanation @netroy!